redaxo/src/addons/backup/composer.json

```
@@ -5,7 +5,7 @@
 
     "config": {
         "platform": {
-            "php": "7.3.0"
+            "php": "8.1.0"
```

hier würde ich die aktuellste php version eintragen, da es ggf. pakete gibt die inbesondere 8.1.0 oder 8.1.1 o.ä. als konflikt haben wg. bugs in den frühen php versionen

```suggestion
            "php": "8.1.13"
```